pkg/operator/osd/osd.go

```diff
@@ -107,8 +107,15 @@ func (c *Cluster) Start() error {
 
 			resources := k8sutil.MergeResourceRequirements(c.Storage.Nodes[i].Resources, c.resources)
 
+			var directories []Directory
+			if len(n.Directories) > 0 {
```

Comment:
could you add a unit test for this override? at a glance, it might be part of the test `TestStorageSpecDevicesAndDirectories`

Reply:
+1